src-electron/ide-integration/studio-rest-api.ts

```diff
@@ -319,6 +323,28 @@ function sendDirtyFlagStatus(db: dbTypes.DbType) {
       })
     })
 }
+async function sendSessionUpgrade(db: dbTypes.DbType) {
+  let sessions = await querySession.getAllSessions(db)
+  for (const session of sessions) {
+    let socket = wsServer.clientSocket(session.sessionKey)
+    let upgrade = await querySession.getSessionUpgradeFlag(
+      db,
+      session.sessionId
+    )
+    let status = await querySession.getSessionUpgradeStatus(
+      db,
+      session.sessionId
+    )
+    if (socket) {
+      if (upgrade == 1) {
+        wsServer.sendWebSocketMessage(socket, {
+          category: dbEnum.wsCategory.upgrade,
+          payload: status,
+        })
```

Comment:
when I merged a line was taken out which changes the UPGRADE entry to 0 - fixing now

Reply:
oh, no. I forgot to update the ts file.